src/gc/env/gcenv.os.h

```diff
@@ -303,6 +303,9 @@ class GCToOSInterface
     //  The number of processors
     static uint32_t GetCurrentProcessCpuCount();
 
+    // Get the budget for the current processor
```

Comment:
This can use better description. `budget for the current processor` is not a well known term.

Reply:
It seems that Quota is a term that would be more well known.